src/ui_ng/lib/src/helm-chart/chart-detail/chart-detail-summary.component.scss

```diff
@@ -0,0 +1,19 @@
+.md-container {
+    margin-top: 15px;
+    border: solid 1px;
+    border-color: #DDDDDD;
+    padding: 3px;
+}
+
+.summary-container {
+    margin-top: 15px;
+
+    table {
+        background-color: #F2F2F2;
+        margin-top: 0.5rem;
```

Comment:
Why some places use rem?Some places use px.

Reply:
table style is originally set by clarity, and its unit is rem;